app/helpers/users_helper.rb

```diff
@@ -1,6 +1,7 @@
 module UsersHelper
-  def gravatar_for user, size: Settings.size_gravatar
+  def gravatar_for user, options = {size: Settings.size_gravatar}
```

Comment:
what is the difference between  def gravatar_for user, options = {size: Settings.size_gravatar} and  def gravatar_for user, options

Reply:
Với def gravatar_for user, options thì phải truyền tham số vào.
Còn def gravatar_for user, options = {size: Settings.size_gravatar} khi không truyền tham số, thì nhận giá trị default là size_gravatar.